speech.yaml

```diff
@@ -209,12 +209,13 @@ components:
             - '0'
             - '1'
         substitutions:
-          type: string
+          type: object
+          additionalProperties:
+            type: string
           description:
-            A set of substitutions to apply to the transcribed text,
-            encoded in a stringified-JSON format of key/value
+            A set of substitutions to apply to the transcribed text, key/value
             (original/substitute) pairs
-          default: '{}'
+          default: {}
```

Comment:
I assume the API has been tested for `{}`?

Reply:
Yes, and is working